### ISSUE TITLE
Document minimum (and max?) Postgres version required

### ISSUE THREAD

**Issue by amercader** [ckan/ckan#5458]
In the CHANGELOG and the install instructions
It should be 9.5, and I'm not sure about the max one. 

See https://github.com/ckan/ckan/pull/5382#issuecomment-646881753

**Comment by WterBerg** [comment]
For what its worth, when I tried to install CKAN 2.8.4 on a Ubuntu 20.04 LTS system I ran into Postgres compatibility issues.
Ubuntu 20.04 apt repositories offer Postgres 12.x. 
Attempting to initialize the CKAN database against that Postgres version resulted in several errors pertaining to `pg_constaint`.

After downgrading to Postgres 11.x no issues were encountered.

To use Postgres 11.x on Ubuntu 20.04:
```
wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
RELEASE=$(lsb_release -cs)
echo "deb http://apt.postgresql.org/pub/repos/apt/ ${RELEASE}"-pgdg main | sudo tee  /etc/apt/sources.list.d/pgdg.list
sudo apt-get update
 
sudo apt-get install -y postgresql-11
```

**Comment by smotornyuk** [comment]
Yep, it's true for CKAN v2.8, because it's using [psycopg2==2.7.3.2](https://github.com/ckan/ckan/blob/2.8/requirements.txt#L33). But CKAN v2.9 will support PostgreSQL v12, because it's using [psycopg2==2.8.2](https://github.com/ckan/ckan/blob/master/requirements.txt#L34)

**Comment by shubham-mahajan** [comment]
@amercader I am using (PostgreSQL) 12.3 with current master and it is working fine and has not faced any issue.
